src/chocolatey.tests/infrastructure.app/commands/ChocolateyPinCommandSpecs.cs

```diff
@@ -412,7 +414,7 @@ public void should_call_nuget_service_list_run_when_command_is_list()
                 configuration.PinCommand.Command = PinCommandType.list;
                 command.run(configuration);
 
-                nugetService.Verify(n => n.list_run(It.IsAny<ChocolateyConfiguration>(), false), Times.Once);
+                nugetService.Verify(n => n.list_run(It.IsAny<ChocolateyConfiguration>(), true), Times.Once);

```

Comment:
Aw foey. You know, I should have double-checked how that was, originally.  In the process of the edits I _removed_ that boolean parameter in favor of the new `config.LogOutput` property, then decided I should do that later, separately.

I think that's going to be a problem, because I'm pretty sure the tests didn't pass with it set to false. I'll investigate.